cli/tests/e2e/snapshots/test_ci/test_dryrun/results.txt

```diff
@@ -63,13 +63,15 @@ First-Party Blocking Rules Fired:
 ┌────────────────┐
 │ Debugging Info │
 └────────────────┘
-Would have sent POST request to create scan
                   
   SCAN ENVIRONMENT
-    versions    - semgrep <MASKED> on python <MASKED>
-    environment - running in environment git, triggering event is unknown  
-    server      - logged in as org_name on https://semgrep.dev             
-               
+  versions    - semgrep <MASKED> on python <MASKED>
+  environment - running in environment git, triggering event is unknown
+            
+  CONNECTION
+Would have sent POST request to create scan
```

Comment:
This indent seems wrong?

Reply:
@brendongo yeah, cause that line is not within rich yet. The design doc asks that we prefix lines like this with

```
! An arbitrary non-fatal error occurred. Here is the warning message.
```

but that's not done yet